client/membership_test.go

```diff
@@ -20,4 +20,7 @@ func TestClient_Members(t *testing.T) {
 	wantEqual(t, got.Items[0].Roles, []types.MembershipRole{types.MembershipRoleCreator})
 	wantNoTimeZero(t, got.Items[0].CreatedAt)
 	wantNoEqual(t, got.Items[0].User, nil)
+	t.Run("pagination", func(t *testing.T) {
+		// TODO: HOW TO CREATE NEW USERS TO TEST THE PAGINATION?
```

Comment:
Oh, that's a good question since you must complete the whole email joining flow 🤔

Reply:
I suggest to leave this one out of the PR and leave the TODO comment to add the tests later. For now just test manually that it works.

Reply:
I agree with this.